mantle.go

```diff
@@ -51,7 +67,7 @@ type Orm struct {
 	Options map[string]string
 }
 
-//mantle is a wrapper for many nosql dbs
+//mantle  wrapper for NoSQL DBs.
 func (o *Orm) New() Mantle {
```

Comment:
Do you want to rename this guy to MantleNosql and then have a Mantle Wrapper which switches between them based on Driver option?